sbt-community-build/src/dotty/communitybuild/sbtplugin/CommunityBuildPlugin.scala

```diff
@@ -0,0 +1,81 @@
+package dotty.communitybuild.sbtplugin
+
+import sbt._
+import sbt.Keys._
+import sbt.librarymanagement.LibraryManagementCodec._
+import sjsonnew.support.scalajson.unsafe.{ Converter, CompactPrinter, Parser }
+import scala.util.{ Try, Failure }
+
+object CommunityBuildPlugin extends AutoPlugin {
```

Comment:
Can you add a quick high-level explanation of what the plugin does as a comment here?